_config.yml

```diff
@@ -150,18 +198,38 @@ secondarynav:
   url: page-components/featured-content-module.html
   internal: true
   category: Page components
+- text: Filterable list control panel
+  url: page-components/filter-control.html
+  internal: true
+  category: Page components
 - text: Form fields
   url: page-components/form-fields.html
   internal: true
   category: Page components
+- text: Full width text
+  url: page-components/fullwidth-text.html
+  internal: true
+  category: Page components
+- text: Half width link blob
+  url: page-components/half-width-link-blob.html
+  internal: true
+  category: Page components
 - text: Links
   url: page-components/links.html
   internal: true
   category: Page components
-# - text: Tables
-#  url: page-components/tables.html
-#  internal: true
-#  category: Page components
+- text: Sidebar/prefooter
+  url: page-components/sidebar-prefooter.html
+  internal: true
+  category: Page components
+- text: Tables
```

Comment:
Actually, I have one request for this PR before merging: Please re-comment out the Tables page in this navigation config. (Comment out lines 225–228.)

The content for this page is still being finished up by @niqjohnson & co.